src/main/java/net/dv8tion/jda/internal/handle/VoiceStateUpdateHandler.java

```diff
@@ -60,6 +70,14 @@ private void handleGuildVoiceState(DataObject content)
         boolean guildDeafened = content.getBoolean("deaf");
         boolean suppressed = content.getBoolean("suppress");
         boolean stream = content.getBoolean("self_stream");
+        String requestToSpeak = content.getString("request_to_speak_timestamp", null);
+        OffsetDateTime requestToSpeakTime = null;
+        long requestToSpeakTimestamp = 0L;
+        if (requestToSpeak != null)
+        {
+            requestToSpeakTime = OffsetDateTime.parse(requestToSpeak);
+            requestToSpeakTimestamp = requestToSpeakTime.toInstant().toEpochMilli();
+        }
```

Comment:
We should have a Helpers method that encompasses this functionality.
We already have a `long -> OffsetDateTime`. We should have one that does `Timestamp(String) -> long`

Reply:
This is not possible here since we want to extract both the OffsetDateTime and a long. Java doesn't support tuple returns so this will have to do.